src/main/java/bio/terra/service/filedata/DrsService.java

```diff
@@ -438,7 +439,7 @@ private DRSObject drsObjectFromFSFile(
       if (passportAuth) {
         accessMethods =
             getDrsSignedURLAccessMethods(
-                ACCESS_ID_PREFIX_GCP + ACCESS_ID_PREFIX_PASSPORT, gcpRegion);
+                ACCESS_ID_PREFIX_GCP + ACCESS_ID_PREFIX_PASSPORT, gcpRegion, true);
```

Comment:
This is purely a personal preference, feel free to disregard.

When I see raw boolean values being passed around, it's harder to tell at a glance what they represent.  In this case, I might pass `passportAuth` to the method instead.

Reply:
I changed this to a raw boolean b/c of an IntelliJ suggestion, but I actually agree - I think using "passportAuth" instead makes it more readable and more robust to future code changes. I'll switch it back! Thanks!